README.md

```diff
@@ -275,6 +275,126 @@ An example policy would look like;
+}
+````
+
+This structure is to ensure that messages that are delivered to SQS queues as a result of directly publishing to the queue have the same format as messages delivered to a queue as a result of being subscribed to an SNS topic to which a message was published. So, from the point of view of a subscriber, whether the message was originally delivered to SNS or SQS is not important.
+
+### Subscribing to SQS
+
+Whether the message was originally published to an SNS topic or sent directly to an SQS queue, the subscriber will receive it by reading from an SQS queue. To do this, JustSaying uses the SQS ReceiveMessage API to receive messages from SQS (documented [here](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_ReceiveMessage.html)). The following parameters are set in JustSaying's request to this API:
+
+#### QueueUrl
+The URL of the queue from which messages are to be received.
+
+#### MaxNumberOfMessages
+JustSaying will vary this value depending on the number of CPU cores available and throttling configuration. 
+
+#### WaitTimeSeconds
+JustSaying sets this to 20.
+
+#### AttributeNames
+JustSaying will only set ApproximateReceiveCount for this parameter.
+
+The response from the API is an object with a single `Messages` property (documented [here](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_Message.html)). The JSON format of each message therein is described [here](https://docs.aws.amazon.com/sns/latest/dg/sns-sqs-as-subscriber.html) and [here](https://docs.aws.amazon.com/sns/latest/dg/sns-message-and-json-formats.html#http-notification-json).
+
+As an example, the following response would be available for that same sample message if it had been published to an SNS topic and then received from an SQS queue subscribed to that topic:
+
+````json
+{
+  "ReceiveMessageResponse": {
+    "ReceiveMessageResult": {
+      "messages": [
+        {
+          "Attributes": null,
+          "Body": "{\n  \"Type\" : \"Notification\",\n  \"MessageId\" : \"redacted\",\n  \"TopicArn\" : \"arn:aws:sns:eu-west-1:redacted:mytopic\",\n  \"Subject\" : \"OrderAccepted\",\n  \"Message\" : \"{\\n    \\\"OrderId\\\": 1234,\\n    \\\"Id\\\": \\\"e3f84a55-b677-43df-8c24-92c170fdd89f\\\",\\n    \\\"TimeStamp\\\": \\\"2019-07-03T09:53:09.2956149Z\\\",\\n    \\\"RaisingComponent\\\": \\\"my sns publisher\\\"\\n}\\n\",\n  \"Timestamp\" : \"2019-07-10T12:38:58.830Z\",\n  \"SignatureVersion\" : \"1\",\n  \"Signature\" : \"redacted\",\n  \"SigningCertURL\" : \"redacted\",\n  \"UnsubscribeURL\" : \"redacted\"\n}",
+          "MD5OfBody": "redacted",
+          "MD5OfMessageAttributes": null,
+          "MessageAttributes": null,
+          "MessageId": "redacted",
+          "ReceiptHandle": "redacted"
+        }
+      ]
+    },
+    "ResponseMetadata": {
+      "RequestId": "redacted"
```

Comment:
I just find `redacted` here an odd choice of word as they aren't secret, they're just superfluous. Maybe just `omitted`?

Reply:
'elided' might be better

Reply:
"Ommitted" sounds OK to me, I'm totally not jumping on that "elide" bandwagon 🤣

I just wanted to remove the original values since they're just giant opaque strings that JustSaying has no influence over and makes no use of so it might look distracting/pointless. I'm in two minds about adding this JSON at all, since it's just copy/paste from the AWS docs and the only thing we care about is a subset of the `Body` property string. Is it useful?

Reply:
I think it's good to see it in raw format to set some context. I had to reverse engineer this for #548, so this would have been useful if it had already existed.

Reply:
Done. All hail the giant JSON blob 🙏